pipeline/tasks/generate-analytics-report.yml

```diff
@@ -0,0 +1,56 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: google/cloud-sdk
+    tag: alpine
+
+
+run:
+  path: sh
+  args:
+    - -exec
+    - |
+
+      apk add --no-progress --update nodejs nodejs-npm
+      echo "npm version"
+      npm -v
+      node --version
+
+      export ROOT_FOLDER=$(pwd)
+      npm config set cache ${ROOT_FOLDER}/.npm --global
+
+      cd census-worth-self-help/analytics
+      npm ci
+
+      echo "REPORTING_MATOMO_BASE_URL: ${REPORTING_MATOMO_BASE_URL}"
+      echo "REPORTING_MATOMO_SITE_ID: ${REPORTING_MATOMO_SITE_ID}"
```

Comment:
Add REPORTING_MATOMO_SITE_NAME to this list of stuff dumped.

Reply:
done